src/pretix/control/templates/pretixcontrol/checkin/reset.html

```diff
@@ -0,0 +1,49 @@
+{% extends "pretixcontrol/items/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% block title %}{% trans "Reset check-in" %}{% endblock %}
+{% block inside %}
+    <h1>{% trans "Reset check-in" %}</h1>
+    <form action="" method="post" class="" data-asynctask>
+        {% csrf_token %}
+        <p>
+            {% blocktrans trimmed %}
+                With this feature, you can reset the entire check-in state of the event.
+                This will delete all check-in records as well as all records of printed tickets or badges.
+                We recommend to use this feature after testing your hardware setup but only before your
+                event started, and you admitted any real attendees or printed any real badges or tickets.
+            {% endblocktrans %}
+        </p>
+        <p>
```

Comment:
I’d make this very prominent:

```suggestion
        <p class="alert alert-danger">
```